pkg/tsdb/elasticsearch/snapshot_test.go

```diff
@@ -169,7 +173,7 @@ func TestResponseSnapshots(t *testing.T) {
 				// we make a copy of the variable to avoid this linter-warning:
 				// "G601: Implicit memory aliasing in for loop."
 				dataResCopy := dataRes
-				experimental.CheckGoldenJSONResponse(t, "testdata_response", goldenFileName, &dataResCopy, false)
+				experimental.CheckGoldenJSONResponse(t, "testdata_response", goldenFileName, &dataResCopy, true)
```

Comment:
This should be turned back to `false` again.

Reply:
whoopsie. very good catch! changed in 93fb67a68583dc494adaa72d0a449d1d87989da7